certmanager.md

```diff
@@ -0,0 +1,178 @@
+
+Certificate management
+======================
+
+Managing certificates with Yunohost
+-----------------------------------
+
+The main feature of the certificate manager is to allow you to install Let's
+Encrypt certificate on your domains without plain. You can use it from the web
+administration (*SSL certificate* on a given domain info page), or from the
+command line with `yunohost domain cert-status`, `cert-install` and
+`cert-renew`.
+
+#### What is required to be able to have a Let's Encrypt certificate ?
+
+Your server needs to be reachable from the rest of Internet on port 80 (and
+443), and make your `domain.tld` points to your server's public IP in your DNS
+configuration. See [this documentation](hdiagnostic) if you need help.
+
+#### Will my certificate be automatically be renewed ?
+
+Yes. Right now, Let's Encrypt certificates are valid 90 days. A cron job will
+run every day and attempt to renew any certificate that will expire in less than
+15 days. An email will be sent to the root user if a renewal fails.
+
+#### I want/need to use a certificate from a different CA than Let's Encrypt.
+
+This is not supported yet. You might need to manually create Certificate Signing
```

Comment:
"This is not supported yet" is too hard : it will make users think it is impossible, while it is actually possible, but with manual expert intervention.